src/AppCommon/Infra/SerializationOptions.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Particular.EndpointThroughputCounter.Infra
+{
+    using System.Text.Encodings.Web;
+    using System.Text.Json;
+    using System.Text.Json.Serialization;
+
+    public static class SerializationOptions
+    {
+        public static readonly JsonSerializerOptions IndentedWithNoEscaping = new()
+        {
+            WriteIndented = true,
+            DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull,
```

Comment:
I had tested this the other day and it worked, but then I got confused how it had worked then, and how would it still work given this change in default condition?

I realized that the attributes were overriding this anyway, as there were no other properties that were nullable that wouldn't be always filled in by the tool. Then I started thinking about each property and null vs. default values and played around a bit more.

Result was a suggestion that couldn't be done in comments so I raised a PR on this PR here: https://github.com/Particular/Particular.EndpointThroughputCounter/pull/759 which will make this class far easier (and more importantly safer) to evolve in the future.

Reply:
@DavidBoike I merged that PR and updated https://github.com/Particular/ParticularTools/pull/375 to match.
Can these PRs be merged now?